docs/examples/101/azure-bastion/main.bicep

```diff
@@ -1,11 +1,30 @@
-param virtualNetworkName string = 'vnet01'
-param vNetIpPrefix string = '10.1.0.0/16'
-param bastionSubnetIpPrefix string = '10.1.1.0/27'
-param bastionHostName string
+@description('Name of new or existing vnet to which Azure Bastion should be deployed')
+param vnet_name string = 'vnet01'
+
+@description('IP prefix for available addresses in vnet address space')
+param vnet_ip_prefix string = '10.1.0.0/16'
+
+@allowed([
+  'new'
+  'existing'
+])
```

Comment:
The pr checklist says params must be camel-cased.  The original quickstart uses hyphens (e.g. "bastion-host-name") which decompiler turns into bastion_host_name, which is what I use here to avoid modifying the original unnecessarily, but maybe we should convert to bastionHostName?  (more work of course)